README.md

```diff
@@ -42,3 +42,15 @@ The grand idea would be to get this repo at least to the level of
 [rules_ros](https://github.com/mvukov/rules_ros). In particular:
 
 - It has to be convenient to cross-compile a ROS2 deployment.
+
+## Alternatives
+
+For alternative approaches, see:
+
+<!--
+DNM: Pending merge of https://github.com/RobotLocomotion/drake-ros/pull/194
```

Comment:
Blocking: Should let this PR merge in, maybe?

Reply:
I think it's just fine to wait for your PR to get merged first and then we make a reference to a main branch of drake-ros below (instead of using a commit hash).

Reply:
Done a41b6b7